dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryAPIImpl.java

```diff
@@ -471,7 +471,7 @@ public void setParents(Categorizable child, List<Category> parents, User user, b
 		LinkedList<Category> children = new LinkedList<Category>(getChildren(category, user, respectFrontendRoles));
 		if (children != null) {
 			while(children.size() > 0) {
-				Category child = children.poll();
+				Category child = (Category) children.poll();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "Category". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)